ui/brave_ads/BUILD.gn

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//build/config/features.gni")
+import("//build/config/ui.gni")
+import("//testing/test.gni")
+import("//ui/base/ui_features.gni")
+
+source_set("brave_ads") {
+  if (is_win || is_linux || is_mac) {
+    deps = [
+      "//base",
+      "//ui/base",
+      "//ui/strings",
+      "//url",
+    ]
+
+    deps += [
+      "//brave/app/theme:brave_theme_resources",
```

Comment:
also anything in `//brave/ui` cannot depend on brave/app, browser/browser, etc.. because it's equivalent to `//ui`